src/Factory/AnnotatedServiceFactory.php

```diff
@@ -19,44 +19,43 @@
 use ReflectionException;
 use ReflectionMethod;
 
-/**
- * Class AnnotatedServiceFactory
- * @package Dot\AnnotatedServiced\Factory
- */
+use function array_shift;
+use function class_exists;
+use function count;
+use function explode;
+use function is_array;
+use function sprintf;
+
 class AnnotatedServiceFactory extends AbstractAnnotatedFactory
 {
     /**
-     * @param ContainerInterface $container
-     * @param $requestedName
      * @return mixed
      * @throws ContainerExceptionInterface
      * @throws NotFoundExceptionInterface
      * @throws ReflectionException
      */
-    public function __invoke(ContainerInterface $container, $requestedName)
+    public function __invoke(ContainerInterface $container, string $requestedName)
     {
         return $this->createObject($container, $requestedName);
     }
 
     /**
-     * @param ContainerInterface $container
-     * @param $requestedName
      * @return mixed
      * @throws ContainerExceptionInterface
      * @throws NotFoundExceptionInterface
      * @throws ReflectionException
      */
-    public function createObject(ContainerInterface $container, $requestedName)
+    public function createObject(ContainerInterface $container, string $requestedName)
```

Comment:
Please remove `return mixed` from the method's docblock and add return type hinting.